pkg/azdiskschedulerextender/main.go

```diff
@@ -93,79 +107,67 @@ func handlePingRequest(response http.ResponseWriter, request *http.Request) {
 }
 
 func handleFilterRequest(response http.ResponseWriter, request *http.Request) {
+	var args schedulerapi.ExtenderArgs
+
+	if request.Body == nil {
```

Comment:
The documentation for [`http.Request`](https://pkg.go.dev/net/http#Request) states that `Body` is non-nil for server requests.